test/comments.test.js

```diff
@@ -1,20 +1,20 @@
-import * as assert from 'proclaim';
+/* eslint-env mocha */
+/* global proclaim sinon */
 import fetchMock from 'fetch-mock';
 import * as fixtures from './helpers/fixtures';
-import sinon from 'sinon/pkg/sinon';
 import Comments from '../src/js/comments';
 import Count from '../src/js/count';
 import Stream from '../src/js/stream';
 
-sinon.assert.expose(assert, {
+sinon.proclaim.expose(proclaim, {
```

Comment:
```suggestion
sinon.assert.expose(proclaim, {
```

https://sinonjs.org/releases/latest/assertions/#sinonassertexposeobject-options